delivery_transsmart/models/res_config.py

```diff
@@ -106,7 +106,10 @@ def get_transsmart_service(self):
         return self.env['delivery.web.service'].browse([wst])
 
     def get_transsmart_carrier_tag(self):
-        return self.env['ir.model.data'].get_object('delivery_transsmart', 'res_partner_category_transsmart_carrier')        
+        #return self.env['ir.model.data'].get_object('delivery_transsmart', 'res_partner_category_transsmart_carrier')        
```

Comment:
Do not leave the commented stuff. We have source management system to see what was there before

Reply:
@NL66278 Yes, missed this one, now removed